pipelines/release/Jenkinsfile.start

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/groovy
+
+/*
+*
+* This script will create a branch out of master branch with name specified in `releaseBranch` parameter variable.
+* Checks for upstream branch with same name; then stops execution with and exception if same branch found in upstream.
+* This Jenkinsfile only support git urls start from `https://`
+*
+* Parameters:
+*   Name:   gitCredentialId
+*      Type:   jenkins parameter; Default value is githubPassword
+*      Description:    contains github username and password for the user to be used
+*   Name:   releaseBranch
+*      Type:   jenkins parameter
+*      Description:    Name of the branch to create
+*
+* Author: Rajesh Rajendran<rjshrjndrn@gmail.com>
+*
+*
+*/
+
+node {
+    // Creating color code strings
+    String ANSI_GREEN = "\u001B[32m"
+    String ANSI_NORMAL = "\u001B[0m"
+    String ANSI_BOLD = "\u001B[1m"
+    String ANSI_RED = "\u001B[31m"
+    def gitCredentialId = params.gitCredentialId ?: 'githubPassword'
```

Comment:
@rjshrjndrn You can remove this. Currently job has no such parameter, so first part is never true. Setup a global variable like private_repo_credentails with id and use that in code.